package.json

```diff
@@ -10,7 +10,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/guppy": "^0.3.5",
+    "@gen3/guppy": "^0.3.6",
```

Comment:
can we pin the guppy to the branch in developing and deploy this portal and guppy to QA to test?

Reply:
done! deployed the guppy feature branch and this branch of data-portal to qa-dcp

Reply:
i added this block to the two explorers in gitops.json, feel free to change and reroll ```"adminAppliedPreFilters": {
      "project_id": {
        "selectedValues": ["jnkns-jenkins"]
      }
    }```